docs/extensions/ScrollViewerExtensions.md

```diff
@@ -7,39 +7,60 @@ keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, ScrollViewer, ext
 
 # ScrollViewer extentions
 
-ScrollViewer extensions provide a simple way to manage Margin for any ScrollBar inside any container.
+The [ScrollViewerExtensions](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.extensions.scrollviewerextensions) provide extension methods to improve your ScrollViewer implementation.
 
-## VerticalScrollBarMargin
+## ScrollBarMargin
 
-The VerticalScrollBarMargin property provides a way to assign a Thickness to the vertical ScrollBar of your container.
+The ScrollBarMargin property provides a way to assign a Thickness to the vertical/horizontal ScrollBar of your container.
+
+### Syntax
+
+```xaml
+<ListView extensions:ScrollViewerEx.HorizontalScrollBarMargin="2, 2, 2, 2">
+    <!-- ListView Item -->
+</ListView>
+
+<ListView extensions:ScrollViewerEx.VerticalScrollBarMargin="2, 2, 2, 2">
+    <!-- ListView Item -->
+</ListView>
+```
+
+### Attached Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| HorizontalScrollBarMargin | Thickness | Set `Thickness` of the horizontal ScrollBar of your container |
+| VerticalScrollBarMargin | Thickness | Set `Thickness` of the vertical ScrollBar of your container |
 
 ### Example
 
 ```xaml
-    <ListView Name="listView"
-              extensions:ScrollViewerExtensions.VerticalScrollBarMargin="{Binding MinHeight, ElementName=MyHeaderGrid, Converter={StaticResource DoubleTopThicknessConverter}}">
-            <ListView.Header>
-                <controls:ScrollHeader Mode="Sticky">
-                    <Grid x:Name="MyHeaderGrid"
-                          MinHeight="100"
-                          Background="{ThemeResource SystemControlAccentAcrylicElementAccentMediumHighBrush}">
-                        <StackPanel HorizontalAlignment="Center"
-                                    VerticalAlignment="Center">
-                            <TextBlock Margin="12"
-                                       FontSize="48"
-                                       FontWeight="Bold"
-                                       Foreground="{StaticResource Brush-White}"
-                                       Text="Scroll Header"
-                                       TextAlignment="Center"
-                                       TextWrapping="WrapWholeWords" />
-                        </StackPanel>
-                    </Grid>
-                </controls:ScrollHeader>
-            </ListView.Header>
-			...
-	</ListView>
+<ListView Name="listView"
+            extensions:ScrollViewerEx.VerticalScrollBarMargin="{Binding MinHeight, ElementName=MyHeaderGrid, Converter={StaticResource DoubleTopThicknessConverter}}">
```

Comment:
Class is still `ScrollViewerExtensions` (as it should be), if you want to abbreviate, I usually make the namespace `ex:` now.  So it'd be `ex:ScrollViewerExtensions.`

Reply:
My bad. But why `ex:ScrollViewerExtensions`? We are using `extensions:ScrollViewerExtensions` in the master branch.